### DISCUSSION TITLE
Video walkthrough?

### DISCUSSION THREAD

**Discussion by 4MAZ** [automatic-ripping-machine/automatic-ripping-machine#618] [Q&A]
I am god awful with Linux but this is an awesome project. Would anyone happen to have a video guide on how to set this up? I have went through the installation multiples times with a fresh install of ubuntu server 20.04 and have had no luck.

**Comment by HyperNerd** [comment]
I made attempts with both Ubuntu, Debian and even with filling in the missing and broken packages didn't have any luck. I have found that docker on Debian yielded the best and most reliable results.
